docs/content/ecs/world-commands.md

```diff
@@ -0,0 +1,124 @@
+        public ComponentDataArray<WorldCommands.CreateEntity.CommandSender> CreateEntitySender;
+    }
+
+    [Inject] Data data;
+
+    protected override void OnUpdate()
+    {
+        for(var i = 0; i < data.Length; i++)
+        {
+            var entity = CreatureTemplate.CreateCreatureEntityTemplate(
+            	new Coordinates(0, 0, 0));
+
+            var request = new WorldCommands.CreateEntity.CreateRequest
+            (
+                entity
+            ));
+
+            data.CreateEntitySender[i].RequestsToSend.Add(request);
+        }
+    }
+}
+```
+
+This system iterates through every entity with a `Foo` component and sends a create entity request.
+
+### 3.  Delete an entity
+
+Deleting entities can be done similarly via the `DeleteEntity` world command. You need to know the SpatialOS entity ID of the entity you want to delete.
+
+To send a request use `WorldCommands.DeleteEntity.CommandSender`. This contains a list of `WorldCommands.DeleteEntity.Request` structs. Add a struct to the list to send the command.
+
+- `TimeoutMillis` is optional.
+
+To receive a response use `WorldCommands.DeleteEntity.CommandResponses`. This contains a list of `WorldCommands.DeleteEntity.ReceivedResponse`.
+
+```csharp
+public class DeleteCreatureSystem : ComponentSystem
+{
+    public struct Data
+    {
+        public readonly int Length;
+        public ComponentDataArray<Bar> Bar;
+        public ComponentDataArray<WorldCommands.DeleteEntity.CommandSender> DeleteEntitySender;
+        [ReadOnly] public ComponentDataArray<SpatialEntityId> SpatialEntityIds;
+    }
+
+    [Inject] Data data;
+
+    protected override void OnUpdate()
+    {
+        for(var i = 0; i < data.Length; i++)
+        {
+        	var entityId = data.SpatialEntityIds[i].EntityId;
+        	
+            var request = new WorldCommands.DeleteEntity.CreateRequest
+            (
+            	entityId
+            ));
+
+            data.DeleteEntitySender[i].RequestsToSend.Add(request);
+        }
+    }
+}
+```
+
+This system iterates through every entity with a `Bar` component and sends a delete entity request.
+
+### 4. Entity query
+
+You can use entity queries to get information about entities in the world. 
+
+To send a request use `WorldCommands.EntityQuery.CommandSender`. This contains a list of `WorldCommands.EntityQuery.Request` structs. Add a struct to the list to send the command
+
+- For more information, see [entity queries](https://docs.improbable.io/reference/latest/shared/glossary#queries) in the SpatialOS documentation.
+- `TimeoutMillis` is optional.
+
+To receive a response use `WorldCommands.EntityQuery.CommandResponses`. This contains a list of `WorldCommands.EntityQuery.ReceivedResponse`.
+
+**Warning**: Entity queries only exist in the GDK in a prototype form, the methods for sending and receiving them exist, but you can not yet safely access the information in the responses. It is not recommended to use them.
```

Comment:
You can access _some_ information in the response. Not sure if we want to be more nuanced